src/libtmux/window.py

```diff
@@ -520,6 +521,10 @@ def split_window(
         if not attach:
             tmux_args += ("-d",)
 
+        if environment:
+            for k, v in environment.items():
+                tmux_args += ("-e%s=%s" % (k, v),)
```

Comment:
tmux 3.0+ compatible (tmux 3.0 compatible: https://github.com/tmux/tmux/blob/42358cc/CHANGES#L823)

Reply:
We could add a version check (and a warning) for tmux 3.0+ (same with the other case + test)

```python
from libtmux.common import has_gte_version

if has_gte_version("3.0"):
```

Reply:
Looks good!